lib/AnsiSerializer.js

```diff
@@ -14,6 +14,7 @@ Object.keys(ansiStyles).forEach(function (styleName) {
 
 function AnsiSerializer(theme) {
     this.theme = theme;
+    this.supportsColor = require('supports-color') || {};
```

Comment:
It would. But the highlighted line is a guard against the case where supports-color is false - there it responds with a falsy value. So because, later on, I referenced a member of it, I decided prudence demanded this guard.